catenax-ng/catenax-ng.github.io
# chore: update docusaurus to version 3.2.1

#### What would be changed or will be added with this PR?
- update docusaurus version
#### Why do i want to change this?
- new release for docusaurus
#### Testing
- checkout this branch
- npm install (optional)
- npm start
- verify changes
#### Preview
![image](https://github.com/catenax-ng/catenax-ng.github.io/assets/121097161/c40ac09c-a7b6-49e2-adbb-ca133f8b49f6)
